install_config/persistent_storage/selector_label_binding.adoc

```diff
@@ -111,6 +111,58 @@ spec:
 labels of targeted PV.
 ====
 
+[[selector-label-volume-bind]]
+=== Optional: Bind the PVC to a specific PV when using a selector
```

Comment:
Is this the right placement for this optional step (between Define and Deploy)? Also, not certain that "when using a selector" is necessary here?

Reply:
The placement is OK, however, the heading does not match the content. Heading says that it's about label selectors, but the two solutions don't use selector at all.

There are two mixed topics:

1. bind a PVC to a specific PV
  * This can be done either by specifying pvc.spec.volumeName (when knowing the PV name) or using pvc.spec.selector (when knowing PV labels). 
2. reserve a PV to specific PVC (or namespace) 
  * This can be done by using a specific storage class or setting pv.spec.claimRef, as outlined correctly below.

I would not mix those in a single chapter.

Reply:
Thanks, @jsafrane. I understand more now that you spelled out #1. 

I've added an additional section for using a selector or volumeName, so there are now two distinct chapters. Let me know if this looks good.